ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -139,48 +144,62 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
 
     @objc public weak var delegate: TwoLineTitleViewDelegate?
 
-    private var alignment: Alignment = .center
+    var currentStyle: Style {
+        didSet {
+            applyStyle()
+        }
+    }
+
+    private lazy var alignmentConstraint: NSLayoutConstraint = centerXAnchor.constraint(equalTo: containingStackView.centerXAnchor)
+    private var alignment: Alignment = .center {
+        didSet {
+            guard alignment != oldValue else {
+                return
+            }
+            alignmentConstraint.isActive = false
+            let keyPath = alignment.xAxisKeyPath
+            alignmentConstraint = self[keyPath: keyPath].constraint(equalTo: containingStackView[keyPath: keyPath])
```

Comment:
Does the `containingStackView` fill the entire view? If it does, just switching between `.center` and `.leading` alignment on the stack should be enough. If it doesn't, maybe it should. Would be a lot cleaner to just change the alignment of the stack instead of messing with constraints every time.

Reply:
Not necessarily. It's possible for `containingStackView` to be smaller than 44px in either dimension.